src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx

```diff
@@ -246,74 +293,32 @@ declare namespace App {
 }
 ```
 
-### 环境变量
+## 平台
 
-请参阅 Cloudflare 的文档了解 [如何使用环境变量](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables)。
+### 标头
 
-```js
-// pages/[id].json.js
-
-export function GET({ params }) {
-  // 在函数内部每个请求都可以访问环境变量
-  const serverUrl = import.meta.env.SERVER_URL;
-  const result = await fetch(serverUrl + "/user/" + params.id);
-  return {
-    body: await result.text(),
-  };
-}
-```
+你可以通过在 Astro 项目的 `public/` 文件夹中添加一个 `_headers` 文件来附加 [自定义头部](https://developers.cloudflare.com/pages/platform/headers/) 到你的响应中，该文件将被复制到构建输出目录中。
 
-### `cloudflare.runtime`
+### 重定向
 
-`runtime: "off" | "local" | "remote"`
-默认值：`"off"`
+你可以使用 Cloudflare Pages 来声明 [自定义重定向](https://developers.cloudflare.com/pages/platform/redirects/)。这允许你将请求重定向到不同的 URL。你可以在 Astro 项目的 `public/` 文件夹中添加一个 `_redirects` 文件，该文件将被复制到构建输出目录。
 
-该可选标志使得 Astro 开发服务器能够填充环境变量和 Cloudflare 请求对象，从而避免使用 Wrangler。
+### 路由
 
-*   `local`：环境变量可用，但请求对象由静态占位符的值填充。
-*   `remote`：环境变量和实时获取的请求对象可用。
-*   `off`：Astro 开发服务器将不会填充环境变量和请求对象。而是使用 Wrangler 来访问 Cloudflare 绑定和环境变量。
+你可以使用 [Cloudflare 路由](https://developers.cloudflare.com/pages/platform/functions/routing/#functions-invocation-routes) 通过 `_routes.json` 文件来定义哪些路由会调用函数，哪些路由是静态资源。该文件由 Astro 自动生成。
 
+#### 自定义 `_routes.json`
 
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import cloudflare from '@astrojs/cloudflare';
-
-export default defineConfig({
-  output: 'server',
-  adapter: cloudflare({
-    runtime: 'off' | 'local' | 'remote',
-  }),
-});
-```
-
-## Wasm 模块导入
-
-`wasmModuleImports: boolean`
-
-默认值：`false`
-
-是否将 `.wasm` 文件[直接作为 ES 模块](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration)导入。
+默认情况下，`@astrojs/cloudflare` 会基于你应用的动态和静态路由生成一个 `_routes.json` 文件，当中包含了 `include` 和 `exclude` 规则。
 
-将 `wasmModuleImports: true` 添加到 `astro.config.mjs` 以在 Cloudflare 构建和 Astro 开发服务器中启用该功能。
 
-```js ins={7}
-// astro.config.mjs
-import {defineConfig} from "astro/config";
-import cloudflare from '@astrojs/cloudflare';
+这将使 Cloudflare 能够在没有函数调用的情况下提供文件和处理静态重定向。创建自定义的 `_routes.json` 将覆盖此自带的优化文件，并且如果没有手动配置，可能导致函数调用次数超过 Cloudflare 计划的请求数限制。
```

Comment:
> This will enable Cloudflare to serve files and process static redirects without a function invocation. Creating a custom `_routes.json` will override this automatic optimization. See [Cloudflare's documentation on creating a custom `routes.json`](https://developers.cloudflare.com/pages/platform/functions/routing/#create-a-_routesjson-file) for more details.

```suggestion
这将使 Cloudflare 能够在没有函数调用的情况下提供文件和处理静态重定向。创建自定义的 `_routes.json` 将覆盖此自带的优化文件。参阅 [Cloudflare 关于创建自定义 `routes.json` 的文档](https://developers.cloudflare.com/pages/platform/functions/routing/#create-a-_routesjson-file) 了解更多细节。
```